packages/tinacms/src/index.ts

```diff
@@ -30,7 +30,7 @@ export { TinaCMS as CMS } from './tina-cms'
 /**
  * Tina Sidebar
  */
-export { Tina, TinaProps } from './components/Tina'
+export { Tina, TinaProviderProps as TinaProps } from './components/Tina'
```

Comment:
do we also need to export `TinaProvider` here?

Reply:
Oh shit that was a bad rename